src/hamster/overview.py

```diff
@@ -210,23 +211,31 @@ def set_values(self, values):
         self.height = len(self.values) * 14
         self._max = max(rec[1] for rec in values) if values else 0
 
+    def set_total(self, total):
+        self._total = total
+
     def _draw(self, context, opacity, matrix):
         g = graphics.Graphics(context)
         g.save_context()
         g.translate(self.x, self.y)
 
+        hours = [(value.days * 24.0 + value.seconds / 3600.0 ) for _,value in self.values]
+        total = self._total.days * 24.0 + self._total.seconds / 3600.0
+
         for i, (label, value) in enumerate(self.values):
+            percent = 100.0 * hours[i] / total
```

Comment:
No error checking if `self._total` is not set (i.e. zero); in that case there will be a division by zero error here!